mix.exs

```diff
@@ -33,17 +33,16 @@ defmodule ExImageInfo.Mixfile do
   defp deps do
     [
       {:excoveralls, "~> 0.18", only: :test},
-      {:ex_doc, "~> 0.30", only: :dev},
+      {:ex_doc, "~> 0.37", only: :dev},
       {:inch_ex, "~> 2.0", only: [:dev, :test]},
```

Comment:
No repo updates in last 3 years, CI service is down (although badges work), has some warnings when compiling. I would consider if removing this along this year.